board/prepare.init

```diff
@@ -373,7 +338,11 @@ wb_firstboot()
         done
     }
 
-    touch $flag
+    touch $FIRSTBOOT_FLAG
+    sync
+
+    reboot
```

Comment:
Do we need this reboot now? It looks like it's better to set boot order properly instead of this.

Reply:
reboot is called when the preparation script has finished. It is executed only at the first boot, later it is ignored. At first boot, it is required.